snuba/datasets/storages/errors.py

```diff
@@ -154,27 +150,18 @@
     storage_key=StorageKey.ERRORS,
     storage_set_key=StorageSetKey.EVENTS,
     schemas=StorageSchemas(read_schema=schema, write_schema=schema),
-    table_writer=TableWriter(
+    stream_loader=KafkaStreamLoader(
```

Comment:
nit: Could you please keep the arguments in the same order as they are in the parent class. This helps readability now that there are many of them.